crates/wit-parser/src/resolve.rs

```diff
@@ -2007,7 +2082,16 @@ impl Remap {
             .zip(&spans.exports)
         {
             let stability = item.stability(resolve);
-            if !resolve.include_stability(stability) {
+            if !resolve
+                .include_stability(stability, pkg_id)
+                .with_context(|| {
+                    format!(
+                        "failed to process feature gate for exported item [{}] in package [{}]",
+                        item.name(resolve).unwrap_or("<unknown>"),
```

Comment:
To avoid adding a `name` function for `WorldItem` could this use the `name` that's in scope here instead of the name on the item itself? The `name` should be sufficient for pinpointing the particular item. (e.g. there's `resolve.name_world_key` for this)

Reply:
Ahh thanks, didn't know that -- will replace